src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('table');
+const button = document.querySelector('.container');
```

Comment:
The variable `button` should be named more descriptively, such as `container`, since it selects the entire container, not just a single button.